private_default.h

```diff
@@ -3,7 +3,11 @@
 #define OSM_CONSUMER_KEY "zz16eUusTr4TcVhoKQrgOJaRzZAeI2ktbVx6e3yn"
 #define OSM_CONSUMER_SECRET "7g4XrlhFkiOG1I3Zpqh8ajo94P9cNNXaU2YCMgL7"
 #define MWM_GEOLOCATION_SERVER ""
-#define METASERVER_URL "https://meta.omaps.app/maps"
+#define METASERVER_SCHEME "https"
```

Comment:
Please don't split the URL here, revert it back and process the string in the Linux part as necessary.

Reply:
All right. Regardless of the revert, I think whatever we use for connection testing, which willbe needed to access these URLs, should be also configurable here. I  think itt's is against any maintener and user interest to scatter such configuration across the codebase. What do you think?